gemmbench/gemm_utils.py

```diff
@@ -116,8 +116,37 @@ def generate_mlir(config: GemmConfig):
         return mlir_template_B
     return mlir_template
 
+@dataclass
+class TkTunedConfig:
+    BLOCK_M: int
+    BLOCK_N: int
+    BLOCK_K: int
+    RATIO_M: int
+    RATIO_N: int
+    WAVES_PER_EU: int
+    MMA_UNITS: int
+    SHARED_UNITS: int
+    GLOBAL_UNITS: int
+    DELAY_MMA: int
+    DELAY_SHARED: int
+    DELAY_GLOBAL: int
+
+def get_tk_tuned_config(config: GemmConfig) -> TunedConfig:
```

Comment:
Changing this to TkTunedConfig should progress the CI

Reply:
thanks will fix.